addons/sale_mrp/models/sale.py

```diff
@@ -13,6 +14,23 @@ class SaleOrder(models.Model):
         compute='_compute_mrp_production_count',
         groups='mrp.group_mrp_user')
 
+    @api.depends('picking_ids')
+    def _compute_picking_ids(self):
```

Comment:
Since we always don't want sales to use the pickings from mrp, you may change `picking_ids` to be computed field too, use this method to compute both `picking_ids` and `delivery_count`.

Reply:
@yhu-odoo picking_ids is a normal field right now can we convert it to compute in the **Stable** version?

Reply:
we can do this FP of master